tensorflow_serving/g3doc/serving_config.md

```diff
@@ -140,3 +140,35 @@ the model config on the fly, to assign a label to it
 (can be achieved using
 [HandleReloadConfigRequest](https://github.com/tensorflow/serving/blob/master/tensorflow_serving/apis/model_service.proto#L22)
 RPC endpoint).
+
+# How to Configure for Monitoring
+
+Creating a monitoring config file containing a
+[MonitoringConfig](https://github.com/tensorflow/serving/blob/1.13.0/tensorflow_serving/config/monitoring_config.proto#L17)
+and set its path to --monitoring_config_file. It currently provides a monitoring
```

Comment:
suggestion:

and set `--monitoring_config_file` flag to the path of this file. It currently ...